From b05cf7d5cd3479e6e97e845d22409b91baa17e8b Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Mon, 16 May 2016 15:14:54 -0700
Subject: [PATCH 601/723] Update to 5.8.1-SNAPSHOT on Mon May 16 15:14:10 PDT
 2016

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '335'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5_5.8.x'

Pushed to remote apache	git@github.sf.cloudera.com:Mirrors/apache-hive.git (push)
origin	git@github.sf.cloudera.com:CDH/hive.git (push)
---
 accumulo-handler/pom.xml              |    2 +-
 ant/pom.xml                           |    2 +-
 beeline/pom.xml                       |    2 +-
 cli/pom.xml                           |    2 +-
 common/pom.xml                        |    2 +-
 contrib/pom.xml                       |    2 +-
 hbase-handler/pom.xml                 |    2 +-
 hcatalog/core/pom.xml                 |    2 +-
 hcatalog/hcatalog-pig-adapter/pom.xml |    2 +-
 hcatalog/pom.xml                      |    2 +-
 hcatalog/server-extensions/pom.xml    |    2 +-
 hcatalog/streaming/pom.xml            |    2 +-
 hcatalog/webhcat/java-client/pom.xml  |    2 +-
 hcatalog/webhcat/svr/pom.xml          |    2 +-
 hwi/pom.xml                           |    2 +-
 itests/custom-serde/pom.xml           |    2 +-
 itests/hcatalog-unit/pom.xml          |    2 +-
 itests/hive-jmh/pom.xml               |    2 +-
 itests/hive-minikdc/pom.xml           |    2 +-
 itests/hive-unit-hadoop2/pom.xml      |    2 +-
 itests/hive-unit/pom.xml              |    2 +-
 itests/pom.xml                        |    2 +-
 itests/qtest-spark/pom.xml            |    2 +-
 itests/qtest/pom.xml                  |    2 +-
 itests/test-serde/pom.xml             |    2 +-
 itests/util/pom.xml                   |    2 +-
 jdbc/pom.xml                          |    2 +-
 metastore/pom.xml                     |    2 +-
 odbc/pom.xml                          |    2 +-
 packaging/pom.xml                     |    2 +-
 pom.xml                               |    6 +++---
 ql/pom.xml                            |    2 +-
 serde/pom.xml                         |    2 +-
 service/pom.xml                       |    2 +-
 shims/0.20S/pom.xml                   |    2 +-
 shims/0.23/pom.xml                    |    2 +-
 shims/aggregator/pom.xml              |    2 +-
 shims/common/pom.xml                  |    2 +-
 shims/pom.xml                         |    2 +-
 shims/scheduler/pom.xml               |    2 +-
 spark-client/pom.xml                  |    2 +-
 testutils/pom.xml                     |    2 +-
 42 files changed, 44 insertions(+), 44 deletions(-)

diff --git a/accumulo-handler/pom.xml b/accumulo-handler/pom.xml
index 448f045..61b7727 100644
--- a/accumulo-handler/pom.xml
+++ b/accumulo-handler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/ant/pom.xml b/ant/pom.xml
index 72bc161..1b2f7e6 100644
--- a/ant/pom.xml
+++ b/ant/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/beeline/pom.xml b/beeline/pom.xml
index 8ee2e66..23aa888 100644
--- a/beeline/pom.xml
+++ b/beeline/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/cli/pom.xml b/cli/pom.xml
index 2eab1eb..c3d9e0c 100644
--- a/cli/pom.xml
+++ b/cli/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/common/pom.xml b/common/pom.xml
index c27f113..275e392 100644
--- a/common/pom.xml
+++ b/common/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/contrib/pom.xml b/contrib/pom.xml
index f17f852..dc7e91b 100644
--- a/contrib/pom.xml
+++ b/contrib/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hbase-handler/pom.xml b/hbase-handler/pom.xml
index 862cc69..9500fb9 100644
--- a/hbase-handler/pom.xml
+++ b/hbase-handler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/core/pom.xml b/hcatalog/core/pom.xml
index 0461163..be4c61d 100644
--- a/hcatalog/core/pom.xml
+++ b/hcatalog/core/pom.xml
@@ -26,7 +26,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/hcatalog-pig-adapter/pom.xml b/hcatalog/hcatalog-pig-adapter/pom.xml
index dd39686..1008296 100644
--- a/hcatalog/hcatalog-pig-adapter/pom.xml
+++ b/hcatalog/hcatalog-pig-adapter/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/pom.xml b/hcatalog/pom.xml
index 1b2092c..71ff3bd 100644
--- a/hcatalog/pom.xml
+++ b/hcatalog/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/server-extensions/pom.xml b/hcatalog/server-extensions/pom.xml
index a28f361..fe2cd62 100644
--- a/hcatalog/server-extensions/pom.xml
+++ b/hcatalog/server-extensions/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/streaming/pom.xml b/hcatalog/streaming/pom.xml
index 6b1d68e..cca3f3a 100644
--- a/hcatalog/streaming/pom.xml
+++ b/hcatalog/streaming/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/webhcat/java-client/pom.xml b/hcatalog/webhcat/java-client/pom.xml
index 77711fc..f5d53b9 100644
--- a/hcatalog/webhcat/java-client/pom.xml
+++ b/hcatalog/webhcat/java-client/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/webhcat/svr/pom.xml b/hcatalog/webhcat/svr/pom.xml
index e817dd1..c40ffe8 100644
--- a/hcatalog/webhcat/svr/pom.xml
+++ b/hcatalog/webhcat/svr/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/hwi/pom.xml b/hwi/pom.xml
index e855587..0d8879c 100644
--- a/hwi/pom.xml
+++ b/hwi/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-serde/pom.xml b/itests/custom-serde/pom.xml
index fb29a4c..2476360 100644
--- a/itests/custom-serde/pom.xml
+++ b/itests/custom-serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hcatalog-unit/pom.xml b/itests/hcatalog-unit/pom.xml
index 71063ab..8e421ac 100644
--- a/itests/hcatalog-unit/pom.xml
+++ b/itests/hcatalog-unit/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-jmh/pom.xml b/itests/hive-jmh/pom.xml
index 549dcc0..6f342e8 100644
--- a/itests/hive-jmh/pom.xml
+++ b/itests/hive-jmh/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-minikdc/pom.xml b/itests/hive-minikdc/pom.xml
index 836bcb8..c9bf399 100644
--- a/itests/hive-minikdc/pom.xml
+++ b/itests/hive-minikdc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-unit-hadoop2/pom.xml b/itests/hive-unit-hadoop2/pom.xml
index d545f28..e6120c5 100644
--- a/itests/hive-unit-hadoop2/pom.xml
+++ b/itests/hive-unit-hadoop2/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-unit/pom.xml b/itests/hive-unit/pom.xml
index d3e53a7..8b3ab7e 100644
--- a/itests/hive-unit/pom.xml
+++ b/itests/hive-unit/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/pom.xml b/itests/pom.xml
index d9a36fd..978ed96 100644
--- a/itests/pom.xml
+++ b/itests/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/qtest-spark/pom.xml b/itests/qtest-spark/pom.xml
index 233321a..ef96d51 100644
--- a/itests/qtest-spark/pom.xml
+++ b/itests/qtest-spark/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/qtest/pom.xml b/itests/qtest/pom.xml
index 2ce459f..dfeadb5 100644
--- a/itests/qtest/pom.xml
+++ b/itests/qtest/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/test-serde/pom.xml b/itests/test-serde/pom.xml
index db82ece..776edad 100644
--- a/itests/test-serde/pom.xml
+++ b/itests/test-serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/util/pom.xml b/itests/util/pom.xml
index c6e9aa9..7b4d5ec 100644
--- a/itests/util/pom.xml
+++ b/itests/util/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/jdbc/pom.xml b/jdbc/pom.xml
index 98ec378..ebdf850 100644
--- a/jdbc/pom.xml
+++ b/jdbc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/metastore/pom.xml b/metastore/pom.xml
index 135b41a..a82e3ec 100644
--- a/metastore/pom.xml
+++ b/metastore/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/odbc/pom.xml b/odbc/pom.xml
index b36118d..e119746 100644
--- a/odbc/pom.xml
+++ b/odbc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/packaging/pom.xml b/packaging/pom.xml
index 208e192..00cb4ce 100644
--- a/packaging/pom.xml
+++ b/packaging/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/pom.xml b/pom.xml
index 73cd873..dc9d2e9 100644
--- a/pom.xml
+++ b/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.8.0-SNAPSHOT</version>
+    <version>5.8.1-SNAPSHOT</version>
   </parent>
 
   <groupId>org.apache.hive</groupId>
   <artifactId>hive</artifactId>
-  <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+  <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <name>Hive</name>
@@ -54,7 +54,7 @@
   </modules>
 
   <properties>
-    <cdh.hive.version>1.1.0-cdh5.8.0-SNAPSHOT</cdh.hive.version>
+    <cdh.hive.version>1.1.0-cdh5.8.1-SNAPSHOT</cdh.hive.version>
     <hive.version.shortname>1.1.0</hive.version.shortname>
 
     <!-- Build Properties -->
diff --git a/ql/pom.xml b/ql/pom.xml
index de11873..91657ef 100644
--- a/ql/pom.xml
+++ b/ql/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/serde/pom.xml b/serde/pom.xml
index 457d5e9..c07960c 100644
--- a/serde/pom.xml
+++ b/serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/service/pom.xml b/service/pom.xml
index e4a793e..06ab785 100644
--- a/service/pom.xml
+++ b/service/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/0.20S/pom.xml b/shims/0.20S/pom.xml
index 86d7e11..207314e 100644
--- a/shims/0.20S/pom.xml
+++ b/shims/0.20S/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/0.23/pom.xml b/shims/0.23/pom.xml
index c364354..90e40d9 100644
--- a/shims/0.23/pom.xml
+++ b/shims/0.23/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/aggregator/pom.xml b/shims/aggregator/pom.xml
index e7c3c9e..679feac 100644
--- a/shims/aggregator/pom.xml
+++ b/shims/aggregator/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/common/pom.xml b/shims/common/pom.xml
index 3094909..15a2a8d 100644
--- a/shims/common/pom.xml
+++ b/shims/common/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/pom.xml b/shims/pom.xml
index c500ab0..697c486 100644
--- a/shims/pom.xml
+++ b/shims/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/scheduler/pom.xml b/shims/scheduler/pom.xml
index 21f511b..7eeb7f5 100644
--- a/shims/scheduler/pom.xml
+++ b/shims/scheduler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/spark-client/pom.xml b/spark-client/pom.xml
index f00db3e..865cedc 100644
--- a/spark-client/pom.xml
+++ b/spark-client/pom.xml
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
   </parent>
 
   <groupId>org.apache.hive</groupId>
diff --git a/testutils/pom.xml b/testutils/pom.xml
index 9b0f0ce..cbab275 100644
--- a/testutils/pom.xml
+++ b/testutils/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.8.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
-- 
1.7.9.5

